nexus-sdk-js/src/Project/index.ts

```diff
@@ -97,7 +97,13 @@ export default class Project {
 
   async getResource(id: string): Promise<Resource> {
     try {
-      return await Resource.getSelf(id, this.orgLabel, this.label);
+      const wildcardSchemaId = '_';
```

Comment:
Why not make this one a global constant on the top-level module scope so that other apps can import it?

Reply:
great idea